src/WrapperApp/components/Results/ResultsTable.tsx

```diff
@@ -28,7 +28,7 @@ const columns: GridColDef[] = [
 		headerName: 'Value',
 		type: 'number',
 		flex: 1,
-		valueGetter: (params: GridValueGetterParams) => formatValue(params.row.value)
+		valueGetter: (_value, row) => formatValue(row.value)
```

Comment:
Everything seems alright according to migration docs but i don't understand this underscore next to 'value' parameter. Is it to make it different from row.value?

Reply:
Quoting gemini:

```
The underscore for _value is a convention to signal that this argument is not essential for the current function's logic. It's a way to improve code readability by highlighting that this parameter is being ignored
```

Reply:
Thanks